src/Telnyx.net/Entities/Calls/CallControlApplications/AnchorsiteOverride.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Newtonsoft.Json;
```

Comment:
Can we move this file into a more common folder? Looks like this and WebHookEnum will be used in more than one service (working on connections now). I was going to put it under Entities -> Enum. Be sure to change the namespace

Reply:
Ok. moved to entities > Enum